x-pack/legacy/plugins/watcher/public/sections/watch_edit/components/json_watch_edit/json_watch_edit_form.tsx

```diff
@@ -208,10 +209,7 @@ export const JsonWatchEditForm = () => {
             </EuiButton>
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <EuiButtonEmpty
-              data-test-subj="btnCancelWatch"
-              href={'#/management/elasticsearch/watcher/watches'}
-            >
+            <EuiButtonEmpty data-test-subj="btnCancelWatch" onClick={() => goToWatchList()}>
```

Comment:
Small nit: Instead of declaring an anonymous function on each render, it is better to write the `onClick` this way:

```js
onClick={goToWatchList}
```

The same applies below in the threshold watcher.

No need to address this on this PR, just sharing the recommendation 😊